apps/goals/serializers.py

```diff
@@ -1,7 +1,7 @@
-from rest_framework_json_api import serializers
+from rest_framework import serializers
```

Comment:
Are we not moving away form json API?

Reply:
No, there was no difference in using this. 

The goal behind the change is to put json:api logic in [here](https://github.com/Real-Dev-Squad/backend-goals/blob/58cb3e0ac88d02c0eed881fae00f5120259181a7/config/settings/base.py#L153-L189) as much as possible. So that we can migrate to other adapters easily in future if needed.